types/unzip/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for unzip 0.1
+// Project: https://github.com/EvanOxfeld/node-unzip
+// Definitions by: Ravi L. <https://github.com/coding2012>
```

Comment:
Ravi L. (coding2012) or ravihugo?

Reply:
Not sure what I'm supposed to change here, I would like to keep it under coding2012 if possible.

Reply:
Just wanted to make sure the mismatch was intentional. 👍